### ISSUE TITLE
Expect error 101 from nextest when build fails?

### ISSUE THREAD

**Issue by sourcefrog** [sourcefrog/cargo-mutants#461]
```
 WARN nextest process exited with unexpected code (not TEST_RUN_FAILED) code=101
unviable src/process.rs:78:9: replace Process::start -> Result<Process> with Ok(Default::default()) in 1.6s build
``` 

Probably this is the normal effect of the build failing, and the warning is wrong?

**Comment by jaskij** [comment]
According to [`nextest-metadata`](https://github.com/nextest-rs/nextest/blob/main/nextest-metadata/src/exit_codes.rs) it does indicate a failed build.

In fact, if I'm reading the [README](https://github.com/nextest-rs/nextest/blob/main/nextest-metadata/README.md) correctly, `nextest-metadata` was made specifically for other tools that wish to wrap `cargo-nextest` and you should be using it.

**Comment by sourcefrog** [comment]
Yes, the very small bug here is to match that interpretation and not emit a warning

**Comment by jaskij** [comment]
Speaking of, I also get a warning on nextest exiting with 4 - which seems to indicate no tests run.
